### DISCUSSION TITLE
g-helper for MSI Notebooks

### DISCUSSION THREAD

**Discussion by Magniflex99** [seerge/g-helper#1891] [Ideas]
Lately I found myself having to change Notebooks, and being my main priority, the quality / price ratio, I found a really excellent used solution, both in terms of temperatures and performance, an MSI GE76 Raider 10UG. Unfortunately the first thing I missed the most, is a management software, complete, clean and light as only it has become g-helper. MSI's Dragon Centre, I don't like it, unfortunately has the same problems of use as the ROG Gaming Centre, however MSI's BIOS is much more open to being compatible with third-party programs, for example the switch mode between 60hz and 300hz of my panel, is recognised by g-helper, even in auto mode. I hope I have made myself better understood. So, could there be a possibility of making g-helper compatible with other devices as well?

**Comment by seerge** [comment]
@Magniflex99 hello, i was answering this already https://github.com/seerge/g-helper/discussions/1842 :) i don't have MSI device, so can't help anyhow

App will launch on MSI device tho ) but obviously most of the features won't work

**Reply by Magniflex99** [reply]
> @Magniflex99 hello, i was answering this already https://github.com/seerge/g-helper/discussions/1842 :) i don't have MSI device, so can't help anyhow
> 
> 
> 
> App will launch on MSI device tho ) but obviously most of the features won't work

Thank you for the reply and I apologise for re-proposing the subject. Got it, if you ever manage to get hold of one of those Notebooks, I would be happy to help you develop a port, giving you feedback on it. The fact is that even looking well, there is nothing like it that can replace the management of the device by proprietary software, for MSI, only programs for Linux, but I wouldn't know how to convert them for operation on Windows, I don't have the knowledge.

**Reply by sonofcopper** [reply]
@seerge Hello,

I have been reading the forums for a few months to see if a program like G helper could be made instead of MSI Center. At this point, I even thought of finding the e-mail address of Seerge, who made G Helper, and sending him an e-mail. But I see that he also had the idea of doing something like this. How can we support you in this regard? Because we really need a program like G-Helper, where we can calculate CPU Watt values and many other values such as Undervolt, I can't explain. The best thing about MSI is that you have extreme freedom when it comes to BIOS. Again, Intel can leave you completely free on the XTU side. But MSI Center is a disgrace. I have never seen worse software. Please do a study on this, Sergee!!!!!!
